src/components/levelSelect.tsx

```diff
@@ -0,0 +1,49 @@
+import '../styles/levelSelect.scss';
+import {useState} from 'react';
+import Dropdown, { Option } from 'react-dropdown';
+import { BsArrowUpShort, BsArrowDownShort } from 'react-icons/bs';
+
+
+function LevelSelect(): JSX.Element {
+
+  const pages = ['1', '2', '3', '4', '5', '6', '7', '8', '9'];
+
+  const [currentPage, setCurrentPage] = useState(6 /* change this to current page once set up */);
```

Comment:
nit: generally, we probably wouldn't like putting comments inside of the fuction!
```suggestion
  const [currentPage, setCurrentPage] = useState(6); // change this to current page once set up
```